modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/LinkTag.java

```diff
@@ -38,33 +37,10 @@ public int doStartTag() throws JspException {
 
 		setContainerElement("a");
 
-		if (Validator.isNotNull(_ariaLabel)) {
-			setDynamicAttribute(StringPool.BLANK, "aria-label", _ariaLabel);
-		}
-
 		if (Validator.isNotNull(_href)) {
 			setDynamicAttribute(StringPool.BLANK, "href", _href);
 		}
 
-		if (Validator.isNotNull(_target)) {
```

Comment:
Check the `target` from dynamic attributes, review this whole method, can't remove everything